lib/workload/stateless/stacks/filemanager/filemanager/src/database/aws/ingester_paired.rs

```diff
@@ -0,0 +1,1822 @@
+//! This module handles logic associated with event ingestion using event ingestion logic that
+//! pairs Created and Deleted events.
+//!
+
```

Comment:
This file is mostly a direct copy from the main branch [`ingester.rs`](https://github.com/umccr/orcabus/blob/main/lib/workload/stateless/stacks/filemanager/filemanager/src/database/aws/ingester.rs).